Examples/SnapinsSDKKotlinExample/app/build.gradle

```diff
@@ -20,26 +29,47 @@ android {
         testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
     }
     buildTypes {
-        release {
+        kotlinImplDebug {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+            signingConfig signingConfigs.defaultSigningConfig
+        }
+        javaImplDebug {
             minifyEnabled false
+            debuggable true
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+            signingConfig signingConfigs.defaultSigningConfig
+        }
+    }
+
+    sourceSets {
+        kotlinImplDebug {
+            res.srcDirs = ['main/res'] // FIXME define in one place?
```

Comment:
Let's move `['main/res']` to a variable in the root `build.gradle`.

Reply:
Done